.github/workflows/stale.yml

```diff
@@ -3,15 +3,21 @@ on:
   schedule:
   - cron: '0 */4 * * *'
 
+permissions:
```

Comment:
lets remove this one also - i dont think we need any read permissions as its a public repo

Reply:
Got it. Thanks